tests/base/polynomials_tensor.output

```diff
@@ -7,7 +7,7 @@ DEAL:BDM::p[3] value 0 1  first 0 0 0 0
 DEAL:BDM::p[4] value 0 5  first 0 0 3 0 
 DEAL:BDM::p[5] value 0 9  first 0 0 0 3 
 DEAL:BDM::p[6] value 4 -11  first 4 0 -5 -3 
-DEAL:BDM::p[7] value -11 9  first -5 -3 0 6 
```

Comment:
This output is a bit surprising, but the code has some rounding implemented which causes the asymmetry.